### PR TITLE
Array+SafeIndex

### PR DESCRIPTION
**Pull request by AppleFlash**

### CODE REVIEW

**Review by fkrasnovid — APPROVED** [review]

**Review by vmzhivetyev — APPROVED** [review]

**Review by Rabbit13245 — APPROVED** [review]

**Comment by vmzhivetyev** [comment]
у вас файл не в папке `Sources`

**Review by vmzhivetyev — CHANGES_REQUESTED** [review]
у вас файл не в папке `Sources`

File: Sources/Foundation/Collection/Collection+SafeIndex.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Foundation
```
Comment (tercteberc) [inline Sources/Foundation/Collection/Collection+SafeIndex.swift]: Please, don't create new file. Move into `Collections+` file with `// MARK: - Safe`

Comment (AppleFlash) [reply Sources/Foundation/Collection/Collection+SafeIndex.swift]: I've updated approach at all

**Comment by AppleFlash** [comment]
There is no sense to add ability to safe subscript for all Collection objects such as Dictionary and Set. I've keep support just for String and Array

**Review by tercteberc — APPROVED** [review]

**Review by vmzhivetyev — APPROVED** [review]

**Pull request merged**